proposals/description/fip_105.ts

```diff
@@ -1,17 +1,115 @@
   commands: [
+    /////////  DAI PSM Skimmer   //////////////
     {
       target: 'core',
       values: '0',
       method: 'grantRole(bytes32,address)',
       arguments: ['0x0866eae1216ed05a11636a648003f3f62921eb97ccb05acc30636f62958a8bd6', '{daiFixedPricePSMFeiSkimmer}'],
       description: 'Grant the new DAI PSM Skimmer the PCV_CONTROLLER_ROLE'
+    },
+
+    ////////    DPI LBP    ////////
+    {
+      target: 'dpi',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: ['{dpiToDaiLBPSwapper}', '37888449801955370645659'],
+      description: 'Transfer DPI from DAO timelock to the LBP swapper'
+    },
+    {
+      target: 'compoundDaiPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{dpiToDaiLBPSwapper}', '187947000000000000000000'],
+      description: 'Withdraw DAI from the CompoundPCVDeposit and transfer to the LBP pool'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposit(address)',
+      arguments: ['{dpiToDaiLensDai}'],
+      description: 'Add DAI swapper lens to the CR oracle'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposit(address)',
+      arguments: ['{dpiToDaiLensDpi}'],
+      description: 'Add DPI swapper lens to the CR oracle'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'removeDeposits(address[])',
+      arguments: [
+        [
+          '{rariPool31FeiPCVDepositWrapper}',
+          '{rariPool25FeiPCVDepositWrapper}',
+          '{rariPool9RaiPCVDepositWrapper}',
+          '{aaveRaiPCVDepositWrapper}',
+          '{rariPool19DpiPCVDepositWrapper}',
+          '{liquityFusePoolLusdPCVDeposit}',
+          '{rariPool72FeiPCVDepositWrapper}',
+          '{raiDepositWrapper}',
+          '{dpiDepositWrapper}'
+        ]
+      ],
+      description: 'Remove various empty PCV deposits from CR oracle'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: ['0x471cfe1a44bf1b786db7d7104d51e6728ed7b90a35394ad7cc424adf8ed16816', '{tribalCouncilTimelock}'],
+      description: 'Grant TribalCouncilTimelock SWAP_ADMIN_ROLE so it can initiate the LBP swap'
+    },
+    /////////// Nope DAO /////////
+    {
+      target: 'nopeDAO',
+      values: '0',
+      method: 'setVotingPeriod(uint256)',
+      arguments: ['26585'], // (86400 * 4) / 13 seconds (assumed 13s block time)
+      description: 'Set the voting period for the NopeDAO to 4 days'
+    },
+    ////////  Transfer CREAM to TribalCouncil Multisig  /////////
+    {
+      target: 'cream',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: ['{tribalCouncilSafe}', '31780370000000000000000'],
+      description: 'Transfer CREAM to TribalCouncil multisig where it will then be swapped'
+    },
```

Comment:
Could you also move the `15,000` that are on the `feiDAOTimelock` to the `aaveEthPCVDeposit` ? they are unreachable by the guardian and it's a good occasion to move them back to a PCVDeposit

Reply:
and keep WETH lens on the feiDAOTimelock because WETH might end up there some time again in the future. We remove the RAI and DPI ones from CR because these assets are liquidated out of the PCV, but it's not the case for WETH

Reply:
Makes sense. Have transferred and then deposited it into the PCV deposit. Kept the lens